src/components/JobsPanelParameters/JobsPanelParameters.js

```diff
@@ -15,7 +15,7 @@ const JobsPanelParameters = ({
   const [addNewParameter, setAddNewParameter] = useState(false)
   const [newParameter, setNewParameter] = useState({
     name: '',
-    type: '',
+    type: 'string',
```

Comment:
You should either use id here and get label wherever you need. Or use label here. If you use label here you can't hardcode it.